public/static/app/perf/perf.js

```diff
@@ -550,6 +550,33 @@ mciModule.controller('PerfController', function PerfController(
         }
         setTimeout(function(){drawDetailGraph($scope.perfSample, $scope.comparePerfSamples, $scope.task.id)},0);
 
+        // Get a list of rejected points.
+        var getRejectedPointsQ = Stitch.use(STITCH_CONFIG.PERF).query(function(db) {
+          const query = {
+            project: $scope.task.branch,
+            task: $scope.task.display_name,
+            variant: $scope.task.build_variant,
+            test: new stitch.BSON.BSONRegExp("^(fio|canary|iperf|NetworkBandwidth)"),
```

Comment:
[nit] Would we want to pull the list we are matching against out to somewhere other pieces of code could access it? It seems like it could be something that would be valuable in other places.

Reply:
The follow on ticket EVG-5912 will also need this (or similar functionality). I have started a public/static/app/common/PointsDataService.js for this set of changes.

With your follow on comment about tests, it makes sense to pull this file in here now. It will mean test for new code only, but that is a good start.

Reply:
sounds good.